### PR TITLE
Basic improvements

### PR DESCRIPTION
**Pull request by abhi-824**
Refactoring of code in index.ts will be done in another issue

### CODE REVIEW

File: schema.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
           {
             "name": "email",
             "type": "String",
-            "isUnique": true
+            "defaultValue":"\"example@gmail.com\""
```
Comment (naman114) [inline schema.json:10]: For purposes of demonstration, email can have "isUnique": `true`
We can add "defaultValue": "\"Lorem Ipsum\"" to blog title field

File: src/index.ts [hunk 2]
```diff
@@ -54,6 +66,9 @@ generator client {
     model.generateSchema();
     prismaSchema += model.prismaModel;
   }
+  for (const _enum of enums) {
```
Comment (naman114) [inline src/index.ts:69]: It's better to have enums above model definitions. Could you move loop above?

File: src/utils/ModelClass.ts [hunk 3]
```diff
@@ -55,12 +56,26 @@ export class Model {
 
   private staticFieldConversion() {
     for (const staticField of this.attributes.staticField) {
+      const defaultValue = `@default(${staticField.defaultValue})`;
+      // switch(staticField.type) {
```
Comment (naman114) [inline src/utils/ModelClass.ts:60]: Clean up the comments please

**Review by naman114 — CHANGES_REQUESTED** [review]
- deleted field should be absent from GET api calls
image

File: src/index.ts (hunk 2 above)
Comment (abhi-824) [reply src/index.ts:69]: We can see it in refactoring the code.

File: src/utils/ModelClass.ts (hunk 3 above)
Comment (abhi-824) [reply src/utils/ModelClass.ts:60]: They were intentional

**Review by naman114 — APPROVED** [review]

**Pull request merged**